src/scribe-plugin-link-prompt-command.js

```diff
@@ -30,6 +31,11 @@ define(function () {
           link = passedLink;
         }
 
+        if(!checks.emptyLink(link)) {
```

Comment:
I initially thought of just doing a trim but taking the comment in the code below indicates that it would be worth starting to break out the checking code in a different testable module.